### ISSUE TITLE
"Line chart" which is present under "Basic Usage" section is not keyboard accessible.

### ISSUE THREAD

**Issue by msftedad** [telerik/kendo-react#1741]
## I'm submitting a...
* Bug report 

## Current behavior
"Crash trend and total crashes for selected Namespace" graph is not keyboard accessible.
**Observation:** "Crash trend and total crashes for selected Namespace" graph is accessible through mouse. Tooltip of crash trend appears on mouse hover of every node.

## Expected behavior
"Crash trend and total crashes for selected Namespace" graph should be keyboard accessible while navigating using tab key.

## Minimal reproduction of the problem with instructions
1.  Open URL: [React Charts Library & Line Component - KendoReact Docs & Demos (telerik.com)](https://www.telerik.com/kendo-react-ui/components/charts/series-types/line/)
2. Navigate to the "Basic usage" section using tab key.
3. Observe whether we can be able to access the graph using keyboard tab key or not.

## What is the motivation or use case for changing the behavior?
Users who cannot use a mouse or other pointing devices, such as individuals with motor disabilities, may not be able to interact with or interpret the chart if it's not keyboard accessible.

## Environment
Browser: OS: Windows 11 Version 22H2 (OS Build 22621.2428)
Edge version 118.0.2088.69 (Official build) (64-bit)

System:
- Node version: 16.13.2
- Platform:   Windows

**Attachments:**
<img width="967" alt="MAS2 1 1_Basic usage chart is not keyboard accessible while navigating using tab key" src="https://github.com/telerik/kendo-react/assets/93735775/98b6b3a9-6391-4156-bf49-8b85e286c830">

https://github.com/telerik/kendo-react/assets/93735775/a4c57c3c-bc4e-4af9-ad33-a5b4c30cedd0

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
We have closed our issue as fixed. So closing this.
